src/sage/symbolic/expression.pyx

```diff
@@ -12890,7 +12890,8 @@ cdef class Expression(Expression_abc):
             sage: plot(2*sin, -4, 4)                                                    # needs sage.plot
             Traceback (most recent call last):
             ...
-            TypeError: unsupported operand parent(s) for *: 'Integer Ring' and '<class 'sage.functions.trig.Function_sin'>'
+            TypeError: unsupported operand parent(s) for *:
+            'Integer Ring' and '<class 'sage.functions.trig.Function_sin'>'
 
```

Comment:
What is the principle here?

I think, if a long line (the output) is split to multiple lines, the second and the following lines should be indented by a space. No?

Reply:
I don't do that in error messages

Reply:
OK.